{
  "id": "uno-ooo-parser",
  "version": "0.1.18",
  "libre_office_ver": "7.2",
  "name": "NoEncryptionException",
  "type": "exception",
  "namespace": "com.sun.star.packages",
  "parser_args": {
    "sort": true,
    "long_names": true,
    "remove_parent_inherited": true
  },
  "writer_args": {
    "include_desc": true
  },
  "data": {
    "from_imports": [
      [
        "..uno.exception",
        "Exception",
        "uno_exception_i"
      ]
    ],
    "from_imports_typing": [],
    "extends_map": {
      "com.sun.star.uno.Exception": "uno_exception_i"
    },
    "quote": [],
    "typings": [],
    "requires_typing": false,
    "full_imports": {
      "general": [
        "com.sun.star.uno.Exception"
      ],
      "typing": []
    },
    "name": "NoEncryptionException",
    "imports": [],
    "namespace": "com.sun.star.packages",
    "extends": [
      "com.sun.star.uno.Exception"
    ],
    "desc": [
      "This exception can be thrown in case object is not encrypted one as expected."
    ],
    "url": "https://api.libreoffice.org/docs/idl/ref/exceptioncom_1_1sun_1_1star_1_1packages_1_1NoEncryptionException.html",
    "items": {}
  }
}